terraform/rds.tf

```diff
@@ -48,7 +41,63 @@ resource "aws_secretsmanager_secret_version" "creds" {
   secret_string = random_password.password.result
 }
 
-resource "aws_rds_cluster" "grafana" {
+resource "aws_kms_key" "this" {
+  # https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/kms_key
+  description         = "Key used to encrypt data in grafana database"
+  enable_key_rotation = true
+  multi_region        = true
+
+  tags = var.common_tags
+}
+
+resource "aws_kms_alias" "a" {
+  # https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/kms_alias
+  name          = "alias/${var.common_tags.environment}-${var.common_tags.service}-grafana-kms-key"
+  target_key_id = aws_kms_key.this.key_id
+}
+
+# resource "aws_rds_cluster" "grafana" {
```

Comment:
should removal of RDS be a follow up PR? we would want to avoid this being applied as is

Reply:
yeah good point. I did the removal in dev to test everything end to end including making sure everything works with the old resources deleted, but ill take it out and remove in a subsequent PR

Reply:
done in d0009374285cf403ba48f530e6975468ed362741